execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -197,13 +240,15 @@ impl TransactionGenerator {
 
     pub fn create_seed_accounts(
         &mut self,
+        reader: Arc<dyn DbReader>,
+        num_new_accounts: usize,
         block_size: usize,
         seed_account_balance: u64,
     ) -> Vec<Vec<Transaction>> {
         let mut txn_block = Vec::new();
 
-        let num_seed_accounts = (self.num_accounts / 1000).max(1).min(1000);
-        let seed_accounts_cache = Self::gen_account_cache(num_seed_accounts, true);
+        let num_seed_accounts = (num_new_accounts / 1000).max(1).min(100000);
+        let seed_accounts_cache = Self::gen_seed_account_cache(reader, num_seed_accounts);
```

Comment:
after this shall we skip making the create / mint txns if num_existing_accounts is not 0? Also you need to know the seq numbers on the seed accounts, which I didn't see where you did. Did I miss anything?

Reply:
It's possible that the new # of seed accounts is larger than the old one so we still need to create some seed accounts.
Given the # of seed account won't be a lot, I guess it's ok to re-create and re-mint some existing seed accounts. (we can avoid this if we store the total number of existing seed accounts, but in general I prefer to store less states when possible)

For the sequence number, it's handled in gen_seed_account_cache by reading the storage directly.

Reply:
Yes, it's something like Keep(MoveAbort(...)) and gas is charged.